Sources/_CryptoExtras/MLDSA/MLDSA_boring.swift

```diff
@@ -16,9 +16,8 @@ import Crypto
 import Foundation
 
 @_implementationOnly import CCryptoBoringSSL
-@_implementationOnly import CCryptoBoringSSLShims
 
-/// A lattice-based digital signature algorithm that provides security against quantum computing attacks.
+/// A module lattice-based digital signature algorithm that provides security against quantum computing attacks.
```

Comment:
Do we want `module` here?

Reply:
I added it just to complete the ML-DSA acronym, maybe I could add an hyphen between it and `lattice` to make it clearer